src/form-service.ts

```diff
@@ -534,6 +549,18 @@ async function getFormElementDynamicOptions(
     ),
   )
 
+  // merge the static options with the URL option results
+  for (const {
+    formElementOptionsSetId,
+    formElementDynamicOptionSetEnvironment,
+  } of staticOptionSets) {
+    results.push({
+      ok: true,
+      formElementOptionsSetId,
+      options: formElementDynamicOptionSetEnvironment?.options,
```

Comment:
If there are no options, we should return `ok: false` with a similar error message to above if the URL is not provided.